polyglot-chat-app/src/main/resources/scripts/script.py

```diff
@@ -1,3 +1,44 @@
+ #
+ # Copyright (c) 2012, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
In Python, just add # for each line. Remove # at line 1 and a closing tag ` #/` at line 40.